packages/angular-cli/commands/init.run.ts

```diff
@@ -101,5 +102,8 @@ export default function initRun(commandOptions: any, rawArgs: string[]) {
       if (commandOptions.linkCli) {
         return linkCli.run();
       }
-    });
+    })
+    .then(function () {
```

Comment:
Use the arrow function so you don't have to `bind(this)`, and use ticks like this:

```typescript
this.ui.writeLine(chalk.green(`Project '${packageName}' successfully created.`));
```